ProcessLib/PhaseField/PhaseFieldProcess.h

```diff
@@ -93,13 +103,29 @@ class PhaseFieldProcess final : public Process
     void postTimestepConcreteProcess(GlobalVector const& x,
                                      int const process_id) override;
 
+    void postNonLinearSolverConcreteProcess(GlobalVector const& x, const double t,
+                                     int const processs_id) override;
+
 private:
     PhaseFieldProcessData<DisplacementDim> _process_data;
 
     std::vector<std::unique_ptr<LocalAssemblerInterface>> _local_assemblers;
 
     std::unique_ptr<NumLib::LocalToGlobalIndexMap>
         _local_to_global_index_map_single_component;
+
+    /// Sparsity pattern for the phase field equation, and it is initialized
+    //  only if the staggered scheme is used.
+    GlobalSparsityPattern _sparsity_pattern_with_single_component;
+
+    /// Check whether the process represented by \c process_id is/has
+    /// mechanical process. In the present implementation, the mechanical
+    /// process has process_id == 0 in the staggered scheme.
+    bool hasMechanicalProcess(int const process_id) const
```

Comment:
From the comment, I'd say this function answers the question isMechanicalProcess, not has.
Where is this function used?

Reply:
The function has been removed.